lib/static/components/modals/screenshot-accepter/meta.jsx

```diff
@@ -1,7 +1,7 @@
 import React, {Component, Fragment} from 'react';
 import PropTypes from 'prop-types';
 
-import MetaInfoContent from '../../section/body/meta-info/content';
+import {MetaInfo as MetaInfoContent} from '@/static/new-ui/components/MetaInfo';
```

Comment:
Are you planning to "move everything necessary to new-ui" and then "remove everything else"?

Reply:
Yeah, that's the idea for components. I currently do not touch reducers and store-related logic.